File: src/test/java/com/github/ccguyka/showupdates/DependencyManagementBuilder.java
package com.github.ccguyka.showupdates;

import org.apache.maven.model.Dependency;
import org.apache.maven.model.DependencyManagement;

class DependencyManagementBuilder {

    public static DependencyManagement from(Dependency... dependencies) {
        DependencyManagement dependencyManagement = new DependencyManagement();
        for (Dependency dependency : dependencies) {
            dependencyManagement.addDependency(dependency);
        }
        return dependencyManagement;
    }
}

File: src/test/kotlin/com/github/ccguyka/showupdates/DependencyManagementBuilder.kt
package com.github.ccguyka.showupdates

import org.apache.maven.model.Dependency
import org.apache.maven.model.DependencyManagement

internal object DependencyManagementBuilder {

    @JvmStatic
    fun from(vararg dependencies: Dependency?): DependencyManagement {
        val dependencyManagement = DependencyManagement()
        for (dependency in dependencies) {
            dependencyManagement.addDependency(dependency)
        }
        return dependencyManagement
    }
}
